src/Core/OrganizationFeatures/OrganizationSponsorships/FamiliesForEnterprise/Cloud/ValidateSponsorshipCommand.cs

```diff
@@ -143,27 +143,27 @@ public async Task<bool> ValidateSponsorshipAsync(Guid sponsoredOrganizationId)
 
     private async Task CancelSponsorshipAsync(Organization sponsoredOrganization, OrganizationSponsorship sponsorship = null)
```

Comment:
Do we still need the async keyword since we are commenting the code. and do you think using a block comments will better than the line comments